tests/test_handlers.py

```diff
@@ -35,6 +35,13 @@ def test_all_handlers(
     extraction_config: ExtractionConfig,
     request: pytest.FixtureRequest,
 ):
+    handlers_by_name = {handler.NAME: handler for handler in extraction_config.handlers}
+    if (
+        input_dir.parent.name in handlers_by_name
+        and not handlers_by_name[input_dir.parent.name].is_supported_here()
+    ):
+        pytest.skip(f"Handler '{input_dir.parent.name}' not supported on this platform")
+
```

Comment:
We have a single testing exception, so for now I would add only the below condition, and skip all the other changes in this commit, until we know we will need some more general solution.

```py
    handler_name = input_dir.parent.name
    if (sys.platform, handler_name) == ('darwin', 'partclone'):
        pytest.skip(f"Handler '{handler_name}' not supported on platform '{sys.platform}'")
```

Reply:
This commit = 8a57c5b3c8a00f6be52a46049f0ab3af8ae8a4a6